scripts/dts/gen_defines.py

```diff
@@ -252,6 +252,10 @@ def write_special_props(node):
     write_interrupts(node)
     write_compatibles(node)
 
+    if "fixed-partitions" in node.parent.compats:
+        macro = f"{node.z_path_id}_PARTITION_ID"
```

Comment:
Why not just emit `{node.z_path_id}_DEP_ORDINAL` for every node? Less of a special case that way.

Reply:
I was being conservative at this point.  I'm good w/renaming the macro to `_DEP_ORDINAL`.  I'd like to limit it to `fixed-partitions` for now until we figure out the next phase of DT w/regards to 'struct device'

Reply:
Okay. Reasonable.